docs/labs/sys_admin_1/lab7-software_management.md

```diff
@@ -525,7 +525,7 @@ Assuming you have already uninstalled the `wget` utility from an exercise, we wi
 
     !!! tip
```

Comment:
**!!! tip "Tip"** 
would make translation work easier,
what do you think?

Reply:
@SergeCroise This was discussed some time ago. I believe the consensus was that the admonitions should be left as-is, because most people understood the basic idea of the admonition "color" and importance, and since the text can be translated within the admonition, that this was good enough. That said, it is certainly worth discussing again. I think bring it up in the channel and we can mull it over there.